wiki/FitNesseRoot/HsacExamples/TaggedTestcases.wiki

```diff
@@ -0,0 +1,23 @@
+---
+Suite
+---
+!1 Suite To test running a filtered suite from maven
+
+Running this suite using: 
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DsuiteFilter=runMe -DexcludeSuiteFilter=dontRun
+  * Results in 3 test cases 001, 002 to be executed (all cases that have the tag 'runMe', and don't have the tag 'dontRun')
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DexcludeSuiteFilter=runMe
+  * Results in 3 test cases 003, 004, 006 to be executed (all cases that don't have the tag 'runMe') 
+ * mvn test-compile failsafe:integration-test -DfitnesseSuiteToRun=HsacExamples.TaggedTestcases -DsuiteFilter=runMe
+  * Results in 3 test cases 001, 002, 005 to be executed (all cases that do have the tag 'runMe')
+  
+Passing a comma separated list to the (exlude)suiteFilter is allowed, so we can also use:
```

Comment:
I would like full control over the tests executed, without changing a line of Java code. So yes, I would like and vs or to be controlled by a system property also...